kratos/utilities/time_discretization.h

```diff
@@ -0,0 +1,113 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Philipp Bucher
+//
+
+#if !defined(KRATOS_TIME_DISCRETIZATION_H_INCLUDED )
+#define  KRATOS_TIME_DISCRETIZATION_H_INCLUDED
+
+// System includes
+#include <vector>
+
+// External includes
+
+// Project includes
+
+namespace Kratos {
+namespace TimeDiscretization {
+
+class BDF1
+{
+public:
+    void ComputeBDFCoefficients(const double DeltaTime,
+                                std::vector<double>& rCoefficients) const;
+};
+
+class BDF2
+{
+public:
+    void ComputeBDFCoefficients(const double DeltaTime,
+                                std::vector<double>& rCoefficients) const;
```

Comment:
```suggestion
                                std::array<double, 3>& rCoefficients) const;
```
Similar changes for the others since the size is known at compile time.

Reply:
@msandre you need to pass this to the processinfo as a variable, so i think it must be a variable-size vector

Reply:
actually this is exactly the point where we disagree

I want to keep those classes lightweight, if someone needs the BDF_COEFFICIENTS in the processinfo then they can do it themselves

=> I am rather sure that this won't even be necessary, I think the original reason this was done bcs this is a way to return values from a process